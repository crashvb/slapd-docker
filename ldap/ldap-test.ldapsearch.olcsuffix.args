simple_bind, anonymous=-x
simple_bind, anonymous, TLS requested=-x -Z
simple_bind, anonymous, TLS forced=-x -ZZ
simple_bind, unauthenticated=-x -D $olcRootDN
simple_bind, unauthenticated, TLS requested=-x -D $olcRootDN -Z
simple_bind, unauthenticated, TLS forced=-x -D $olcRootDN -ZZ
simple_bind, authenticated=-x -D $olcRootDN -w $LDAP_ADMIN_PASSWORD
simple_bind, authenticated, TLS requested=-x -D $olcRootDN -w $LDAP_ADMIN_PASSWORD -Z
simple_bind, authenticated, TLS forced=-x -D $olcRootDN -w $LDAP_ADMIN_PASSWORD -ZZ
SASL=-Y EXTERNAL
SASL, TLS requested=-Y EXTERNAL -Z
SASL, TLS forced=-Y EXTERNAL -ZZ
