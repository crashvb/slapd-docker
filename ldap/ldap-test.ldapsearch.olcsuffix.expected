ldap, simple_bind, anonymous=disallowed
ldap, simple_bind, anonymous, TLS requested=disallowed
ldap, simple_bind, anonymous, TLS forced=disallowed
ldap, simple_bind, unauthenticated=disallowed
ldap, simple_bind, unauthenticated, TLS requested=disallowed
ldap, simple_bind, unauthenticated, TLS forced=disallowed
ldap, simple_bind, authenticated=confidentiality required
ldap, SASL=no mechanism available
ldapi, simple_bind, anonymous=disallowed
ldapi, simple_bind, anonymous, TLS requested=disallowed
ldapi, simple_bind, anonymous, TLS forced=disallowed
ldapi, simple_bind, unauthenticated=disallowed
ldapi, simple_bind, unauthenticated, TLS requested=disallowed
ldapi, simple_bind, unauthenticated, TLS forced=disallowed
ldaps, simple_bind, anonymous=disallowed
ldaps, simple_bind, anonymous, TLS requested=already started
ldaps, simple_bind, anonymous, TLS forced=already started
ldaps, simple_bind, unauthenticated=disallowed
ldaps, simple_bind, unauthenticated, TLS requested=already started
ldaps, simple_bind, unauthenticated, TLS forced=already started
ldaps, simple_bind, authenticated, TLS forced=already started
ldaps, SASL, TLS forced=already started
